src/styles/blocks/_about-us.scss

```diff
@@ -0,0 +1,88 @@
+.about-us,
```

Comment:
rename all file in `blocks` folder without `_`.

More about file naming in scss you could watch [in this video](https://mate.academy/learn/css-advanced/sass#/video/932)